src/Cli/dotnet/CommonOptions.cs

```diff
@@ -4,6 +4,7 @@
 using System.CommandLine;
 using System.CommandLine.Completions;
 using System.CommandLine.Parsing;
+using System.ComponentModel.DataAnnotations;
```

Comment:
the tinyist of nits: was this intended? I don't see any usage of DataAnnotations?

Reply:
It wasn't. IDE sometimes adds reandom usings :(